### PR TITLE
Changed UI of Password Change form

### LINKED ISSUE #182: Improve registration templates
Improve the following registration templates in the project:
- [x] Password change form page (Assigned to @malavikarajeshvikraman)
- [x] Password change done page (Assigned to @rray14)
- [x] Password reset form page (Assigned to @aishwarya0816)
- [x] Password reset done page (Assigned to @Fairspy)
- [ ] Password reset confirm page (Assigned to @sk1122)
- [x] Password reset complete page (Assigned to @cryptofox17)

You can refer to these templates while working on parts of this issue: [`signup_success`](https://jagrati.herokuapp.com/accounts/signup/success), [`account_authenticated`](https://jagrati.herokuapp.com/accounts/authenticated), [`token_expired`](https://jagrati.herokuapp.com/accounts/activate/gsg/gsdg/)`.

As the different pages can be worked upon separately, multiple contributors can claim this issue.

### PR DESCRIPTION
**Pull request by malavikarajeshvikraman**
# Related Issue
Fixes:  #182 

# Prosposed Changes
- Changed UI of Password Change form ( fix #182)

# Screenshots

Original             |  Updated
:-------------------------:|:-------------------------:
 
image
 |
image

### CODE REVIEW

File: templates/registration/password_change_form.html [hunk 1]
```diff
@@ -4,9 +4,10 @@
 <style type="text/css">
     .form-signin {
         width: 100%;
-        max-width: 330px;
+        max-width: 450px;
         padding: 15px;
         margin: auto;
+        margin-top: 7.5%;
```
Comment (garg3133) [inline templates/registration/password_change_form.html:10]: Don't enter margin in %. Either use px or em.

File: templates/registration/password_change_form.html [hunk 2]
```diff
@@ -36,7 +37,8 @@
 {% block content %}
 
 <form method="POST" class="form-signin">{% csrf_token %}
-    <h1 class="h3 mb-3 font-weight-normal">Change Password</h1>
+    <i class="fas fa-unlock-alt" style="font-size: 7em; color: orange; margin-left:150px;margin-bottom: 3%;" ></i>
```
Comment (garg3133) [inline templates/registration/password_change_form.html:40]: Don't write so many inline styles. You can create a new class and specify the properties there.

Also, the colour hex should be `#ff8c00`. Again, don't specify margin in %.

File: templates/registration/password_change_form.html [hunk 3]
```diff
@@ -47,7 +49,7 @@ <h1 class="h3 mb-3 font-weight-normal">Change Password</h1>
         {% endfor %}
     {% endfor %}
 
-    <button class="btn btn-lg btn-primary btn-block" type="submit">Submit</button>  
+    <button style = "background-color: orange;color: black;" class="btn btn-lg btn-primary btn-block" type="submit">Submit</button>
```
Comment (garg3133) [inline templates/registration/password_change_form.html:52]: `background-color` should be `#ff8c00`. Also, remove the border around the button.

Plus, specify the css properties by specifying a separate class.

**Review by garg3133 — COMMENTED** [review]
@malavikarajeshvikraman Thanks for your PR. I've requested a few changes below.

Also, the icon is not displayed properly on small screens. Please check that too.

**Comment by malavikarajeshvikraman** [comment]
@garg3133  I have made the changes could you pls check

File: templates/registration/password_change_form.html [hunk 4]
```diff
@@ -29,14 +30,26 @@
...
     }
+    .fas {
+        font-size: 7em; 
+        color: #ff8c00;  
+        margin-bottom: 0.35rm;
+    }
+    .bt{
+        background-color: #ff8c00;
+        color: #000000;
+        border:0px ;
```
Comment (garg3133) [inline templates/registration/password_change_form.html:41]: `border: none` would be a better property.

File: templates/registration/password_change_form.html [hunk 5]
```diff
@@ -47,7 +60,8 @@ <h1 class="h3 mb-3 font-weight-normal">Change Password</h1>
         {% endfor %}
     {% endfor %}
 
-    <button class="btn btn-lg btn-primary btn-block" type="submit">Submit</button>  
+    <button  class="bt btn btn-lg btn-primary btn-block" type="submit">Submit</button>
```
Comment (garg3133) [inline templates/registration/password_change_form.html:63]: Button is getting blue upon hover. maybe this is due to the `btn-primary` class. Check that.

**Comment by malavikarajeshvikraman** [comment]
@garg3133  I have made the changes you have requested. Please do merge if it's alright.

**Comment by garg3133** [comment]
Looks good! Merging it into the master! 🎉 

Thanks, @malavikarajeshvikraman for your contribution 🚀

**Pull request merged**